### PR TITLE
Making conversion command from .fog to .aws/credentials

### PR DESCRIPTION
**Pull request by emmasax1**
Description and Impact
----------------------
Makes a command that will change the `~/.fog` file to `~/.aws/credentials` file automatically.

Deploy Plan
-----------
> Does Platform Operations need to know anything special about this deploy? Are migrations present?

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
> Links to bug tickets or user stories.

QA Plan
-------
- [ ]  Test to make sure that command works correctly by typing `opsicle legacy_credential_converter` from the main directory of opsicle

### CODE REVIEW

File: lib/opsicle/commands/legacy_credential_converter.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+module Opsicle
+  class LegacyCredentialConverter
+    include CredentialConverterHelper
+
+    def initialize(environment)
+      @environment = environment
+    end
+
+    def execute(options={ monitor: true })
```
Comment (anfleene) [inline lib/opsicle/commands/legacy_credential_converter.rb]: you don't need this

**Comment by emmasax1** [comment]
```
>> opsicle
NAME
    opsicle - Opsworks Command Line Utility Belt

    
DOCUMENTATION
    For documentation and help in setting up your configuration files,
    see Opsicle's GitHub repo: https://github.com/sportngin/opsicle


SYNOPSIS
    opsicle [global options] command [command options] [arguments...]

VERSION
    0.18.1

GLOBAL OPTIONS
    --[no-]color - Use colored output (default: enabled)
    --debug      - Enable Debug mode for detailed logs and backtraces
    --help       - Show this message
    --verbose    - Enable Verbose mode for more logging
    --version    - Display the program version

COMMANDS
    chef-update                 - Update the Stack Custom Chef Cookbooks
    deploy                      - Deploy your current app to the given environment stack
    execute-recipes             - Execute arbitrary recipes on the Stack
    help                        - Shows a list of commands or help for one command
    instances                   - List all instances in the given environment stack
    legacy_credential_converter - Convert the ~/.fog file to a ~/.aws/credentials file
    list                        - List all apps in the given environment stack
    monitor                     - Launch the Opsicle Stack Monitor for the given environment stack
    opsworks-url                - Show the OpsWorks URL for the given environment stack
    ssh                         - SSH access to instances in the given environment stack
    ssh-clean-keys              - Clear an environment's keys from your ssh known_hosts file
    ssh-key                     - Set your user SSH key (PUBLIC KEY) for OpsWorks
    update                      - Update properties on a OpsWorks resource.
    user-profile-info           - Display IAM user profile information as JSON.
```

Command running is:
```
>> opsicle legacy_credential_converter
Converting your ~/.fog file to a ~/.aws/credentials file...
```

Result of command is:
.fog:
```
# You may have existing keys here, replace them with the keys you created
# If you have production deploy access you can leave the production section alone
staging:
  aws_access_key_id: ACCESS_KEY_ID
  aws_secret_access_key: SECRET_ACCESS_KEY
```
.aws/credentials:
```
[staging]
aws_access_key_id = ACCESS_KEY_ID
aws_secret_access_key = SECRET_ACCESS_KEY
```

**Comment by anfleene** [comment]
QA and CR are good 👍

**Comment by emmasax1** [comment]
:octocat: Merged into deployable.2016.05.18.

**Pull request merged**